enable/container.py

```diff
@@ -199,8 +199,10 @@ def compact(self):
             # Update our position and the positions of all of our components,
             # but do it quietly
             for component in self._components:
-                component.set(position = [component.x-ll_x, component.y-ll_y],
-                              trait_change_notify = False)
+                component.trait_set(
+                    position=[component.x-ll_x, component.y-ll_y],
+                    trait_change_notify=False
```

Comment:
How do you feel about replacing `.trait_set(..., trait_change_notify=False)` by `.trait_setq(...)`?

Reply:
Yeah. That would be a bit better, I think.